src/workerd/api/global-scope.h

```diff
@@ -303,6 +304,10 @@ class ServiceWorkerGlobalScope: public WorkerGlobalScope {
       jsg::V8Ref<v8::Promise> promise,
       jsg::Value value);
 
+  typedef kj::Function<jsg::Ref<Socket>(jsg::Lock&)> ConnectFn;
+  void setConnectOverride(kj::String networkAddress, ConnectFn connectFn);
+  kj::Maybe<ConnectFn&> getConnectOverride(kj::StringPtr networkAddress);
```

Comment:
Nit: Some code comments for these would be helpful

Reply:
Good callout, thanks. Done.